cmd/cdi-controller/controller.go

```diff
@@ -130,9 +128,12 @@ func init() {
 		klog.V(1).Infof("Note: increase the -v level in the controller deployment for more detailed logging, eg. -v=%d or -v=%d\n", 2, 3)
 	}
 
-	// Register metrics for our various controllers
-	metrics.Registry = prometheus.NewRegistry()
-	registerMetrics()
+	// Setup metrics for our various controllers
+	if err := metrics.SetupMetrics(); err != nil {
+		klog.Errorf("failed to setup metrics: %v", err)
+		os.Exit(1)
+	}
+	metrics.SetIncompleteProfileValue(-1)
```

Comment:
use functions with more meaning instead of receiving a magic value as argument

Reply:
removed the "Value" from the func name and parameters